test/fixtures/changelog/activeadmin.md

```diff
@@ -0,0 +1,429 @@
+[#5453]: https://github.com/activeadmin/activeadmin/pull/5453
+[#5458]: https://github.com/activeadmin/activeadmin/pull/5458
+[#5461]: https://github.com/activeadmin/activeadmin/pull/5461
+[#5464]: https://github.com/activeadmin/activeadmin/pull/5464
+[#5486]: https://github.com/activeadmin/activeadmin/pull/5486
+[#5501]: https://github.com/activeadmin/activeadmin/pull/5501
+[#5517]: https://github.com/activeadmin/activeadmin/pull/5517
+[#5537]: https://github.com/activeadmin/activeadmin/pull/5537
+[#5583]: https://github.com/activeadmin/activeadmin/pull/5583
+
+[@5t111111]: https://github.com/5t111111
+[@aarek]: https://github.com/aarek
+[@ajw725]: https://github.com/ajw725
+[@alex-bogomolov]: https://github.com/alex-bogomolov
+[@andreslemik]: https://github.com/andreslemik
+[@blocknotes]: https://github.com/blocknotes
+[@bolshakov]: https://github.com/bolshakov
+[@chancancode]: https://github.com/chancancode
+[@chumakoff]: https://github.com/chumakoff
+[@craigmcnamara]: https://github.com/craigmcnamara
+[@DanielHeath]: https://github.com/DanielHeath
+[@deivid-rodriguez]: https://github.com/deivid-rodriguez
+[@dennisvdvliet]: https://github.com/dennisvdvliet
+[@dhyegofernando]: https://github.com/dhyegofernando
+[@dmitry]: https://github.com/dmitry
+[@drn]: https://github.com/drn
+[@eikes]: https://github.com/eikes
+[@f1sherman]: https://github.com/f1sherman
+[@faucct]: https://github.com/faucct
+[@Fivell]: https://github.com/Fivell
+[@glebtv]: https://github.com/glebtv
+[@gonzedge]: https://github.com/gonzedge
+[@innparusu95]: https://github.com/innparusu95
+[@jasl]: https://github.com/jasl
+[@javierjulio]: https://github.com/javierjulio
+[@jawa]: https://github.com/jawa
+[@JiiHu]: https://github.com/JiiHu
+[@johnnyshields]: https://github.com/johnnyshields
+[@kjeldahl]: https://github.com/kjeldahl
+[@kobeumut]: https://github.com/kobeumut
+[@leio10]: https://github.com/leio10
+[@markstory]: https://github.com/markstory
+[@mauriciopasquier]: https://github.com/mauriciopasquier
+[@mconiglio]: https://github.com/mconiglio
+[@Nguyenanh]: https://github.com/Nguyenanh
+[@PChambino]: https://github.com/PChambino
+[@potatosalad]: https://github.com/potatosalad
+[@pranas]: https://github.com/pranas
+[@renotocn]: https://github.com/renotocn
+[@RobinvanderVliet]: https://github.com/RobinvanderVliet
+[@rogerkk]: https://github.com/rogerkk
+[@seanlinsley]: https://github.com/seanlinsley
+[@ShallmentMo]: https://github.com/ShallmentMo
+[@shekibobo]: https://github.com/shekibobo
+[@shouya]: https://github.com/shouya
+[@stefsava]: https://github.com/stefsava
+[@tiagotex]: https://github.com/tiagotex
+[@timoschilling]: https://github.com/timoschilling
+[@TimPetricola]: https://github.com/TimPetricola
+[@varyonic]: https://github.com/varyonic
+[@wasifhossain]: https://github.com/wasifhossain
+[@Wowu]: https://github.com/Wowu
+[@wspurgin]: https://github.com/wspurgin
+[@zorab47]: https://github.com/zorab47
+[@chrp]: https://github.com/chrp
```

Comment:
This is part of a real changelog, that's why it's quite big. But if you want a more minimal file, I can reduce it.